core/src/main/java/com/linecorp/armeria/server/HttpServerCodec.java

```diff
@@ -0,0 +1,171 @@
+package com.linecorp.armeria.server;
```

Comment:
Missing copyright https://armeria.dev/community/developer-guide#add-copyright-header

Also, can you copy over the original license?
https://github.com/line/armeria/blob/1def296db95a8d5a2dfdf71600e621db5c649f7a/core/src/main/java/com/linecorp/armeria/server/Http2ServerUpgradeCodec.java#L16-L29